nibabel/openers.py

```diff
@@ -117,6 +130,9 @@ class Opener(object):
     default_compresslevel = 1
     #: whether to ignore case looking for compression extensions
     compress_ext_icase = True
+    #: hint which tells us whether the file handle will be kept open for
+    #  multiple reads/writes, or just for one-time access.
+    default_keep_open = False
```

Comment:
Is this class variable intended to be changed by a power user? Under what circumstances might I want to change that?

Reply:
I'm not actually sure - I was just going with the convention set from the other kwargs. I can't really think of a use-case for changing it, so will change it default it to `False` in `__init__`.